sdk-base/src/main/java/com/mapbox/maps/plugin/scalebar/ScaleBarPlugin.kt

```diff
@@ -12,4 +13,13 @@ interface ScaleBarPlugin : ViewPlugin, MapSizePlugin, ScaleBarSettingsInterface
    * How many meters in each pixel.
    */
   var distancePerPixel: Float
+
+  /**
+   * If set to True scale bar will be triggering onDraw depending on [ScaleBarSettings.refreshInterval]
+   * even if actual data did not change. If set to False scale bar will redraw only on demand.
+   *
+   * Defaults to False and should not be changed explicitly in most cases.
+   * Could be set to True to produce correct GPU frame metrics when running gfxinfo command.
+   */
+  var useContinuousRendering: Boolean
```

Comment:
Looks like these two options both can be generated through serialisation spec generator, wdyt?

Reply:
I was thinking about it but found several points against:
1. This API is public but rather debug one and we do not want encourage user to use it that badly.
2. Putting it to style spec will also make iOS implement it (in theory) but this option seems to be very Android specific.

Reply:
> This API is public but rather debug one and we do not want encourage user to use it that badly.

Well splitting the public API surface also doesn't sounds right 😄 We intend to ask users to adjust the settings using scaleBar.updateSettings { }, but these APIs won't be there..  I prefer the generated approach since it will offer consistent API surface, and easier to manager our public API surfaces.

> Putting it to style spec will also make iOS implement it (in theory) but this option seems to be very Android specific.

We have option to mark it as an Android-only property in the spec 😏